src/System.Private.CoreLib/src/System/Decimal.DecCalc.cs

```diff
@@ -71,7 +73,7 @@ internal static void DecMul10(ref Decimal value)
         /// <summary>
         /// Class that contains all the mathematical calculations for decimal. Most of which have been ported from oleaut32.
         /// </summary>
-        private class DecCalc
+        private static unsafe class DecCalc
```

Comment:
It would nice to remove `unsafe` from here and instead add `unsafe` explicitly to the few places where the unsafe code is used, so that they are easy to look for.